CONTRIBUTING.md

```diff
@@ -101,15 +101,19 @@ The following project structure should be followed:
 - Authorization header: `Authorization: Bearer <token>`
 
 ## Production login:
+
 https://github.com/login/oauth/authorize?client_id=23c78f66ab7964e5ef97
```

Comment:
all the login URLs can now be changed to `<protocol>://<domain_name>/auth/github/login
eg: `http://localhost:300/auth/github/login` or `https://api.realdevsquad.com/auth/github/login`

This is  now possible with reference to this PR: https://github.com/Real-Dev-Squad/website-backend/pull/840